### PR TITLE
caddy: download to random tempdir, and use the caddy_file variable

### PR DESCRIPTION
**Pull request by vbatts**
Signed-off-by: Vincent Batts <vbatts@hashbangbash.com>

### CODE REVIEW

File: roles/caddy/tasks/main.yml [hunk 1]
```diff
@@ -34,8 +34,7 @@
 
 - name: install caddy
   shell: |
-    mkdir -p /tmp/caddy-{{ caddy_version }}
-    cd /tmp/caddy-{{ caddy_version }}
```
Comment (vbatts) [inline roles/caddy/tasks/main.yml:38]: I can see we you thankfully removed the renaming of the file to the version, but still this variable could be out of sync with the binary downloaded, so just download to a random tmp.

File: roles/caddy/tasks/main.yml [hunk 2]
```diff
@@ -46,7 +45,7 @@
 
 - name: install caddyfile
   template:
-    src: Caddyfile
+    src: "{{ caddy_file }}"
```
Comment (vbatts) [inline roles/caddy/tasks/main.yml:48]: without this variable, it was pulling it just the template/Caddyfile, not the local variable we were using

Comment (schu) [reply roles/caddy/tasks/main.yml:48]: Indeed, this is meant to be configurable.

File: roles/caddy/tasks/main.yml (hunk 1 above)
Comment (schu) [reply roles/caddy/tasks/main.yml:38]: 👍

**Review by schu — APPROVED** [review]
Thanks Vincent!

**Pull request merged**